contrib/core-contract-tests/tests/pox-4/pox-4.prop.test.ts

```diff
@@ -0,0 +1,1838 @@
+            [Cl.principal(caller)],
+            caller
+          );
+          // Assert
+          assert(isClarityType(actual, ClarityType.OptionalNone));
+        }
+      )
+    );
+  });
+
+  it("should return none get-delegation-info", () => {
+    fc.assert(
+      fc.property(
+        fc.constantFrom(...simnet.getAccounts().values()),
+        (caller) => {
+          // Arrange
+          // Act
+          const { result: actual } = simnet.callReadOnlyFn(
+            "pox-4",
+            "get-delegation-info",
+            [Cl.principal(caller)],
+            caller
+          );
+          // Assert
+          assert(isClarityType(actual, ClarityType.OptionalNone));
+        }
+      )
+    );
+  });
+
+  it("should return correct get-pox-info", () => {
+    fc.assert(
+      fc.property(
+        fc.constantFrom(...simnet.getAccounts().values()),
+        (caller) => {
+          // Arrange
+          const expected_reward_cycle_id = 0,
+            expected_first_burn_block_height = 0;
+          // Act
+          const { result: actual } = simnet.callReadOnlyFn(
+            "pox-4",
+            "get-pox-info",
+            [],
+            caller
+          );
+          // Assert
+          assert(isClarityType(actual, ClarityType.ResponseOk));
+          assert(isClarityType(actual.value, ClarityType.Tuple));
+          expect(actual.value.data["first-burnchain-block-height"]).toBeUint(
+            expected_first_burn_block_height
+          );
+          expect(actual.value.data["min-amount-ustx"]).toBeUint(
+            MIN_AMOUNT_USTX
+          );
+          expect(actual.value.data["prepare-cycle-length"]).toBeUint(
+            TESTNET_PREPARE_CYCLE_LENGTH
+          );
+          expect(actual.value.data["reward-cycle-id"]).toBeUint(
+            expected_reward_cycle_id
+          );
+          expect(actual.value.data["reward-cycle-length"]).toBeUint(
+            TESTNET_REWARD_CYCLE_LENGTH
+          );
+          expect(actual.value.data["total-liquid-supply-ustx"]).toBeUint(
+            INITIAL_TOTAL_LIQ_SUPPLY
+          );
+        }
+      )
+    );
+  });
+
+  it("should return none get-allowance-contract-caller", () => {
+    fc.assert(
+      fc.property(
+        fc.constantFrom(...simnet.getAccounts().values()),
+        fc.constantFrom(...simnet.getAccounts().values()),
+        fc.constantFrom(...simnet.getAccounts().values()),
+        (caller, sender, contract_caller) => {
+          // Arrange
+          // Act
+          const { result: actual } = simnet.callReadOnlyFn(
+            "pox-4",
+            "get-allowance-contract-callers",
+            [Cl.principal(sender), Cl.principal(contract_caller)],
+            caller
+          );
+          // Assert
+          assert(isClarityType(actual, ClarityType.OptionalNone));
+        }
+      )
+    );
+  });
+
+  it("should return some(until-burn-ht: none) get-allowance-contract-caller after allow-contract-caller", () => {
```

Comment:
This is failing for me locally. Seeing a "TypeError" issue: Object.entries(...).toSorted is not a function.

Reply:
Perhaps Node.js/npm/TypeScript issue 🤔

Reply:
It is the same the CI is throwing. Not sure how to handle it or where it is coming from. No error for me locally.

Reply:
Just pushed an update. I have added another assertion before the failing expect. Hope this will solve the error.

Reply:
Thanks! However this still seem to fail locally. What's your environment? (Node.js version?)

Reply:
This looks like a bug in the assertion library, it's thrown here:

```
    1519|           expect(actual.value).toBeTuple({ "until-burn-ht": Cl.none() });
       |                                ^
```

@hugocaillard, perhaps you have an idea?

In case it helps, the dependencies used are:

```
    "@hirosystems/clarinet-sdk": "^1.1.0",
    "@stacks/transactions": "^6.12.0",
    "chokidar-cli": "^3.0.0",
    "@stacks/clarunit": "0.0.1",
    "fast-check": "^3.15.1",
    "typescript": "^5.2.2",
    "vite": "^4.4.9",
    "vitest": "^0.34.4",
    "vitest-environment-clarinet": "^1.0.0"
```

Here's the strack trace:

```
Got TypeError: Object.entries(...).toSorted is not a function
    at Object.toBeTuple (contrib/core-contract-tests/node_modules/@hirosystems/clarinet-sdk/vitest-helpers/src/clarityValuesMatchers.ts:232:54)
    at Proxy.expectWrapper (contrib/core-contract-tests/node_modules/@vitest/expect/dist/index.js:1368:40)
    at Proxy.<anonymous> (contrib/core-contract-tests/node_modules/@vitest/expect/dist/index.js:653:17)
    at Proxy.methodWrapper (contrib/core-contract-tests/node_modules/chai/lib/chai/utils/addMethod.js:57:25)
    at contrib/core-contract-tests/tests/pox-4/pox-4.prop.test.ts:1519:32
    at Property.predicate (contrib/core-contract-tests/node_modules/fast-check/lib/esm/check/property/Property.js:14:54)
    at Property.run (contrib/core-contract-tests/node_modules/fast-check/lib/esm/check/property/Property.generic.js:46:33)
    at runIt (contrib/core-contract-tests/node_modules/fast-check/lib/esm/check/runner/Runner.js:18:30)
    at check (contrib/core-contract-tests/node_modules/fast-check/lib/esm/check/runner/Runner.js:62:11)
    at Module.assert (contrib/core-contract-tests/node_modules/fast-check/lib/esm/check/runner/Runner.js:65:17)
```